### PR TITLE
Issue #689 Fix text about vm that provisions openshift

### LINKED ISSUE #689: Fix hypervisor requirement text in Getting Started and README
Following the work on PR #624, @LalatenduMohanty gave feedback that the section describing the VM needs to be adjusted. See comment https://github.com/minishift/minishift/pull/624#pullrequestreview-30055851

At second glance, I noticed that this text also appears in the README, so will fix it in both locations.

### PR DESCRIPTION
**Pull request by thatdocslady**
Fixes #689 

This PR updates the text in the hypervisor prerequisites to reflect how the VM and OpenShift interact, per @LalatenduMohanty feedback on PR #624 .

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -30,9 +30,9 @@ https://ci.centos.org/job/minishift/[image:https://ci.centos.org/buildStatus/ico
 To download the latest binary and review release notes, see
 the https://github.com/minishift/minishift/releases[Minishift releases] page.
 
-Minishift requires a hypervisor to start the virtual machine containing
-OpenShift. Make sure that the hypervisor of your choice is installed and
-enabled on your system before you install Minishift.
+Minishift requires a hypervisor to start the virtual machine that provisions OpenShift.
+Make sure that the hypervisor of your choice is installed and enabled on your system
+before you install Minishift.
```
Comment (praveenkumar) [inline README.adoc]: @thatdocslady `before you install minishift` => `before you start minishift` ?

File: docs/source/getting-started/installing.adoc [hunk 2]
```diff
@@ -15,9 +15,9 @@ The following section describes how to install Minishift and the required depend
 [[install-prerequisites]]
 == Prerequisites
 
-Minishift requires a hypervisor to start the virtual machine containing
-OpenShift. Make sure that the hypervisor of your choice is installed and
-enabled on your system before you install Minishift.
+Minishift requires a hypervisor to start the virtual machine that provisions OpenShift.
+Make sure that the hypervisor of your choice is installed and enabled on your system
+before you install Minishift.
```
Comment (praveenkumar) [inline docs/source/getting-started/installing.adoc]: same as above.

File: README.adoc (hunk 1 above)
Comment (thatdocslady) [inline README.adoc]: @praveenkumar AFAIK it's before you install, not before you start. @hferentschik ?

Comment (hferentschik) [inline README.adoc]: +1, really there is no install process per se.

File: docs/source/getting-started/installing.adoc (hunk 2 above)
Comment (hferentschik) [inline docs/source/getting-started/installing.adoc]: +1

**Comment by thatdocslady** [comment]
@praveenkumar @hferentschik fixed the text to also reflect the install/start question

File: README.adoc [hunk 3]
```diff
@@ -30,9 +30,9 @@ https://ci.centos.org/job/minishift/[image:https://ci.centos.org/buildStatus/ico
 To download the latest binary and review release notes, see
 the https://github.com/minishift/minishift/releases[Minishift releases] page.
 
-Minishift requires a hypervisor to start the virtual machine containing
-OpenShift. Make sure that the hypervisor of your choice is installed and
-enabled on your system before you install Minishift.
+Minishift requires a hypervisor to start the virtual machine that provisions OpenShift.
```
Comment (hferentschik) [inline README.adoc]: This seems to say that the VM is provisioning OpenShift. That's not correct. 

"Minishift requires a hypervisor to start the virtual machine on/onto which OpenShift is provisioned"

**Comment by thatdocslady** [comment]
@hferentschik @praveenkumar fixed the text per feedback.

**Pull request merged**